ledger-core/v2/cmd/insolard_headless/main.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	jww "github.com/spf13/jwalterweatherman"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/log/global"
+	"github.com/insolar/assured-ledger/ledger-core/v2/server"
+	"github.com/insolar/assured-ledger/ledger-core/v2/version"
+)
+
+func main() {
```

Comment:
Headless network is not a separate binary, it's a mode of insolard binary. See AppController doc

Reply:
my task is named as "headless network in SEPARATE binary", so I separated them. If it needed to be only a mode I can change this behavior